Templates/templates/PHP/Model/EntityType.php.tt

```diff
@@ -223,7 +223,7 @@ foreach(var property in entity.Properties.Where(prop => prop.Type.GetTypeString(
     public function set<#=propertyName.ToCheckedCase()#>($val)
     {
 <#
-    if (propertyTypeString == "\\DateTime") {
+    if (propertyTypeString == "\\DateTime" || propertyTypeString == "\\Microsoft\\Graph\\Core\\Models\\TimeOfDay" || propertyTypeString == "\\Microsoft\\Graph\\Core\\Models\\Date" ) {
```

Comment:
Our custom `TimeOfDay` and `Date` types don't support `format(\DateTime::ISO8601)`
Also not sure where this section of the template is used. Can't see any model that uses it.